ui/app/models/task.js

```diff
@@ -21,15 +21,15 @@ export default class Task extends Fragment {
   @fragmentArray('action', { defaultValue: () => [] })
   actions;
 
-  @attr() meta;
+  @fragment('structured-attributes') meta;
 
   @fragment('task-schedule') schedule;
 
-  @computed('taskGroup.mergedMeta', 'meta')
+  @computed('meta.raw', 'taskGroup.mergedMeta')
   get mergedMeta() {
     return {
       ...this.taskGroup.mergedMeta,
-      ...this.meta,
+      ...this.meta?.raw,
```

Comment:
You might notice that this is using the `...this.meta.raw` format where the above task-group model is using `this.get('meta.raw')`. The `get` syntax is valid JS and everything, but is more of an old-ember convention, and task group models have existed longer than task models in our codebase.

You can see these old-ember ways of doing things scattered throughout: look for a `@classic` decorator near the top of the class declaration.